.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close Stale Issues'
+on:
+  schedule:
+    - cron: '32 11 * * 1-5'  # 11:32 UTC / 07:32 EDT / 06:32 EST from Mon-Fri.
+
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30  # Don't wait 6 hours to cancel workflow if it hangs.
+    steps:
+      - uses: actions/stale@v5
+        with:
+          stale-issue-label: 'stale'
+          stale-issue-message: |
+            This issue has been automatically marked as stale because it has not had any recent activity. It will be closed if no further activity occurs but can be reopened if it becomes relevant again.
```

Comment:
Nit: Should this mention the 90/30?

LGTM either wait.

Reply:
```suggestion
            This issue has been automatically marked as stale because it had no activity for 90 days. It will be closed if no activity occurs in the next 7 days but can be reopened if it becomes relevant again.
```